tests/foreman/ui/test_discoveredhosts.py

```diff
@@ -97,7 +110,7 @@ def test_host_discovery(self):
         @Assert: Host should be successfully discovered
 
         """
-        mac = gen_mac(multicast=True, locally=True)
+        mac = gen_mac(multicast=False, locally=True)
```

Comment:
Sure.. actually we are running tests on external libvirt and there behaviour got changed.. Tests were failing due to below error:

```
robottelo.ssh: DEBUG: <<< stderr
ERROR    XML error: expected unicast mac address, found multicast '03:18:29:2b:ec:e0'
```
So I set the multicast to `False`